tss-esapi/src/context/tpm_commands/testing.rs

```diff
@@ -33,7 +33,11 @@ impl Context {
 
     /// Get the TPM self test result
     ///
+    /// # Details
     /// The returned buffer data is manufacturer-specific information.
+    ///
+    /// TpmFormatZeroResponseCode::Error(TpmFormatZeroError::Failure) may be returned by TPM2_GetTestResult()
+    /// as the 'testResultparameter'.
```

Comment:
:thinking: to be honest this is hard to parse but frankly, the function signature doesn't look nice too: `Result<(MaxBuffer, Result<()>)>`.

Maybe it'd be a good idea to linkify `TpmFormatZeroResponseCode::Error(TpmFormatZeroError::Failure)` (or at least add backticks).

Why mention `TPM2_GetTestResult()` here? Because it's an underlying routine? Maybe using `get_test_result` would be better?

What's `'testResultparameter'`?

Reply:
Fixed but please check again if the new version is better.

Reply:
I think it looks very nice :ok_hand: thanks!